### PR TITLE
Final

### PR DESCRIPTION
**Pull request by calgray**
I think it might be a good idea this gets merged back to master before submission.

💯/💯

### CODE REVIEW

File: olsr-routing-protocol.cc [hunk 1]
```diff
@@ -942,6 +943,7 @@ RoutingProtocol::MprComputation ()
       std::clog << mprvector[i] << ",";
     }
     std::clog << "]" << std::endl;
+    */
```
Comment (judilsteve) [inline olsr-routing-protocol.cc:946]: Don't commit commented code

File: olsr-routing-protocol.cc [hunk 2]
```diff
@@ -1820,11 +1833,34 @@ RoutingProtocol::SendTc ()
   tc.ansn = m_ansn;
   
   //TODO: MODIFICATION 3: Randomly remove half the elements from MprSelectorSet
-  for (MprSelectorSet::const_iterator mprsel_tuple = m_state.GetMprSelectors ().begin ();
-       mprsel_tuple != m_state.GetMprSelectors ().end (); mprsel_tuple++)
-    {
-      tc.neighborAddresses.push_back (mprsel_tuple->mainAddr);
+  
+   if(m_mode == 3) {
```
Comment (judilsteve) [inline olsr-routing-protocol.cc:1837]: This would be much easier to follow if m_mode were an enum

File: project2-olsr.cc [hunk 3]
```diff
@@ -53,35 +59,34 @@ WifiPhyStandard phyStandard = WIFI_PHY_STANDARD_80211b;
...
 
-bool enableFlowmon 	= true;
+bool outputFlowmon         = false;
 
-bool enableNetanim 	= false;
-bool netanimCounters 	= false;
-bool packetMetadata	= true;
+bool enableNetanim         = false;
+bool netanimCounters     = false;
+bool packetMetadata        = true;
```
Comment (judilsteve) [inline project2-olsr.cc:89]: These variables should be static const and defined inside a namespace

File: olsr-routing-protocol.cc (hunk 2 above)
Comment (calgray) [reply olsr-routing-protocol.cc:1837]: I hope I don't have to to write much network interference code again

File: olsr-routing-protocol.cc (hunk 1 above)
Comment (calgray) [reply olsr-routing-protocol.cc:946]: I wonder what it does 🤔

File: project2-olsr.cc (hunk 3 above)
Comment (calgray) [reply project2-olsr.cc:89]: It looks like we forgot to pick a project name for this all together

**Comment by calgray** [comment]
I question if I can be bothered to get this to compile again as I think this was written on linux. I'll save this until I decide to duelboot again

**Pull request merged**